spec/models/notification_service/slack_service_spec.rb

```diff
@@ -12,6 +12,15 @@
   let(:room_id) do
     "#general"
   end
+  let(:backtrace_lines) do
```

Comment:
Could this please just be an array of strings to test against? That way you avoid duplicating the implementation. 

What do you think?

Reply:
I'll give it a try.

Reply:
@rud The backtrace is always different.  What strings do you want me to use?  They have to be determined by whatever Fabricate throws at us.

Reply:
Maybe test for one or more interesting substrings you would expect to see in the output? Something that wasn't there before would be optimal. If the factory is giving you too random data, an idea is usually to more tightly control a few of the things it generates. 

Sorry if that's too abstract, please let me know and I'll help take a look at the concrete behaviour. Deal?

Reply:
Do you want me to edit bracktrace_fabricator.rb so it's not random?

Reply:
I can do something like

Reply:
Yeah, wouldn't that make sense in this case?